tests/backup/backup_rbac_test.go

```diff
@@ -0,0 +1,501 @@
+package tests
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/pborman/uuid"
+	api "github.com/portworx/px-backup-api/pkg/apis/v1"
+	"github.com/portworx/torpedo/drivers"
+	"github.com/portworx/torpedo/drivers/backup"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+)
+
+// VerfiyRBACforInfraAdmin Validates the RBAC opeartion for infra-admin user.
+var _ = Describe("{VerfiyRBACforInfraAdmin}", func() {
```

Comment:
nit: change `Verfiy` to `Verify` everywhere

Reply:
Done